edx_name_affirmation/templates/edx_name_affirmation/base.html

```diff
@@ -0,0 +1,22 @@
+{% raw %}
```

Comment:
Don't believe we want to keep any of the HTML templates around. We want to keep this library largely backend, right?

Reply:
Yeah, I'll clean those up.